source/includes/entityref/paybill/_clientcorporationrateagreementcardlinegroup.md

```diff
@@ -31,49 +31,73 @@ Represents a group of rates on a Client Corporation
         <tr class="odd">
             <td>clientCorporationRateAgreementCard</td>
             <td>To-one association</td>
-            <td>ClientCorporationRateAgreementCard</td>
+            <td>ClientCorporationRateAgreementCard
+                <p>Default fields:</p>
+                <ul>
+                    <li>id</li>
+                    <li>effectiveDate</li>
+                    <li>effectiveEndDate</li>
+                </ul>
+            </td>
             <td></td>
             <td>X</td>
         </tr>
         <tr class="even">
             <td>clientCorporationRateAgreementCardLines</td>
             <td>To-many association</td>
-            <td>ClientCorporationRateAgreementCardLine</td>
+            <td>ClientCorporationRateAgreementCardLine - (No more than there are EarnCodes for the given EarnCodeGroup)
+                <p>Default fields:</p>
+                <ul>
+                    <li>id</li>
+                </ul>
+            </td>
             <td></td>
             <td>X</td>
         </tr>
         <tr class="odd">
             <td>clientCorporationRateAgreementCardVersion</td>
             <td>To-one association</td>
-            <td>ClientCorporationRateAgreementCardVersion</td>
+            <td>ClientCorporationRateAgreementCardVersion
+                <p>Default fields:</p>
+                <ul>
+                    <li>id</li>
+                    <li>effectiveDate</li>
+                    <li>effectiveEndDate</li>
+                </ul>
+            </td>
             <td>X</td>
             <td>X</td>
         </tr>
         <tr class="even">
             <td>earnCodeGroup</td>
             <td>To-one association</td>
-            <td>EarnCodeGroup</td>
+            <td>Earn Code Group for all of the rates
+                <p>Default fields:</p>
+                <ul>
+                    <li>id</li>
+                </ul>
+            </td>
             <td></td>
             <td>X</td>
         </tr>
         <tr class="odd">
             <td>externalID</td>
             <td>String (100)</td>
-            <td></td>
+            <td>External identifier for the record, used for migrations and back-office Integration.</td>
             <td></td>
             <td>X</td>
         </tr>
         <tr class="even">
             <td>isBase</td>
             <td>Boolean</td>
-            <td></td>
+            <td>This is the list of default rates for the rate card.</td>
             <td>X</td>
             <td>X</td>
         </tr>
         <tr class="odd">
             <td>migrateGUID</td>
             <td>String (36)</td>
-            <td></td>
+            <td>The Migrate GUID.</td>
```

Comment:
See other pages that have this and match. I think they have something about it being unused.
Same in other instances of this.